.github/workflows/operatorPRBuildAndDeploy.yml

```diff
@@ -47,7 +47,34 @@ jobs:
           export PATH=$GOROOT/bin:$GOPATH/bin:$PATH
           export MO_TAGS=$(cat ./build/tagsToPush.txt)
           mage operator:deployToECR
-      - name: Deploy to GH Packages
+      - name: Deploy to GH Packages - standard image
+        if: github.event.pull_request.head.repo.full_name == 'datastax/cass-operator'
+        env:
+          MO_GH_USR: 'datastax/cass-operator'
+          MO_GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          MO_GH_PKG_REPO: 'datastax/cass-operator/operator'
+        run: |
+          export PATH=$GOROOT/bin:$GOPATH/bin:$PATH
+          export MO_TAGS=$(cat ./build/tagsToPush.txt)
+          mage operator:deployToGHPackages
+      - name: Build docker - ubi image
```

Comment:
kind of meh to have these 3 steps duplicated for the UBI stuff. The alternative would be to add an extra env var/logic to the mage lib to support "build and tag both in one command". I could go either way on that.... but it would just be more conditionals and env stuff to keep track of

Reply:
Actions does have a way to loop over things if that would be neater